Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 0
	at QuRe.main(QuRe.java:17)
------------------------------------------------------
